pyproject.toml

```diff
@@ -33,6 +33,7 @@ dependencies = [
     "rdkit",
     "scikit-learn",
     "scipy",
+    "tensorboard",
```

Comment:
where in the code do we use `tensorboard`? It's a sub-dep of `lightning`, but this file only enumerates the libraries that are _directly_ needed.

Reply:
actually you're right. It looks like it's not installed, but is needed if we're using a `TensorboardLogger` in the CLI.